airbyte-webapp/src/components/connectorBuilder/StreamTestingPanel/ConfigMenu.tsx

```diff
@@ -36,13 +38,18 @@ export const ConfigMenu: React.FC<ConfigMenuProps> = ({ className }) => {
     <>
       <Tooltip
         control={
-          <Button
-            size="sm"
-            variant="secondary"
-            onClick={() => setIsOpen(true)}
-            disabled={!jsonManifest.spec}
-            icon={<FontAwesomeIcon className={styles.icon} icon={faGear} />}
-          />
+          <>
+            <Button
+              size="sm"
+              variant="secondary"
+              onClick={() => setIsOpen(true)}
+              disabled={!jsonManifest.spec}
+              icon={<FontAwesomeIcon className={styles.icon} icon={faGear} />}
+            />
+            {configJsonErrors > 0 && (
+              <NumberBadge className={styles.inputsErrorBadge} value={configJsonErrors} color="red" />
```

Comment:
If you hover over an element with the inspect tool, it tells you the contrast ratio, that's what I'm using to check whether it will be readable on all kinds of devices:

<img width="321" alt="Screenshot 2022-12-21 at 10 39 16" src="https://user-images.githubusercontent.com/1508364/208873417-8ac80c40-5bb9-4ce2-906c-1dfecbd80982.png">
<img width="335" alt="Screenshot 2022-12-21 at 10 39 26" src="https://user-images.githubusercontent.com/1508364/208873421-de0b854c-9b0a-4832-bb6e-ac421831f86b.png">

> Maybe we can make the red color darker here or something to achieve that?

That would work but I would like to split this out from this PR as the NumberBadge is a standard component used in a bunch of places and I would like to keep consistency.

> Also, using a number badge here makes me think: should we also display the number of errors in each view in the view red dot indicators on the builder left sidebar

Would be a nice follow-up.

Reply:
Created an issue: https://github.com/airbytehq/airbyte/issues/20768